rocAL/rocAL/include/api/rocal_api_augmentation.h

```diff
@@ -143,7 +143,7 @@ extern "C"  RocalImage  ROCAL_API_CALL rocalRotateFixed(RocalContext context, Ro
 extern "C" RocalTensor ROCAL_API_CALL rocalBrightness(RocalContext context, RocalTensor input, bool is_output,
                                                       RocalFloatParam alpha = NULL, RocalFloatParam beta = NULL,
                                                       RocalTensorLayout rocal_tensor_output_layout = ROCAL_NHWC,
-                                                      RocalTensorOutputType rocal_tensor_output_type = ROCAL_UINT8);
+                                                      RocalTensorOutputType rocal_tensor_output_dtype = ROCAL_UINT8);
```

Comment:
both looks exactly identical. Can you please revert if no change

Reply:
reverted